handlebars/src/test/java/com/github/jknack/handlebars/AbstractTest.java

```diff
@@ -87,7 +87,7 @@ public void shouldCompileTo(final String template, final Object context,
       throws IOException {
     Template t = compile(template, helpers, partials);
     String result = t.apply(configureContext(context));
-    assertEquals("'" + expected + "' should === '" + result + "': " + message, expected, result);
+    assertEquals("'" + result + "' should === '" + expected + "': " + message, expected, result);
```

Comment:
Minor fix to the order of arguments so that `result should equal expected` instead of the other way around